trussworks/react-uswds
# chore(deps): update various dependencies

# Summary

This PR fixes a typo in the `CODEOWNERS` file and upgrades various deps listed below

- Storybook releated deps
- Danger
- Happo.io
- Husky
- webpack-cli

## How To Test

Ensure build works `yarn`
Ensure tests run `yarn test`
Run storybook `yarn storybook`
